P202.Training.Test/RoleServiceTest.cs

```diff
@@ -0,0 +1,139 @@
+﻿using AutoMapper;
+using Moq;
+using P202.Training.Data.Entities;
+using P202.Training.Data.Repositories;
+using P202.Training.Domain;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace P202.Training.Test
+{
+    public class RoleServiceTest
+    {
+        [Fact]
+        public void CreateRole_PersistsChange()
+        {
+            // Arrange
+            var roleRepositoryMock = new Mock<IRoleRepository>();
+            var mappingServiceMock = new Mock<IMapper>();
```

Comment:
minor: rename to `mapperMock` so it's easier to identify which class we are mocking

Reply:
Done!